src/django/api/management/commands/populate_tigerline_data.py

```
@@ -0,0 +1,68 @@
+        try:
+            self.stdout.write(f'Processing CSV file: {s3_key}...')
+            populate_tigerline_data(
+                USCountyTigerline,
+                s3_key,
+                source_srid=source_srid,
+                clear_existing=clear_existing
+            )
+
+            if clear_existing:
+                self.stdout.write(
+                    self.style.SUCCESS(f'Cleared existing records')
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)